.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   validate:
     executor:
       name: hmpps/localstack
-      jdk_tag: "17.0"
+      jdk_tag: "18.0"
```

Comment:
With the upgrade to 18 you need to comment out the daily `hmpps/veracode_pipeline_scan:` line 110 to 113 as veracode don't support java18 yet